mw-server/db/scripts/testdata/testdata.go

```diff
@@ -0,0 +1,188 @@
+VALUES
+('8563a330-eedb-4e71-9abc-4db6fbabd13b', 'this is finished job description', 1000, '1b3d5e7f-5a1e-4d3a-b1a5-d1a1d5b7a7e1', '25ceb64e-7a57-4ce0-a4fd-45982d9fce38');
+
+INSERT INTO "job_dones_job_tags" ("job_done_uuid", "job_tag_uuid")
+VALUES
+('8563a330-eedb-4e71-9abc-4db6fbabd13b', 'c73ff20b-e64e-4e5f-b270-1a40ba1bd81b');
+
+INSERT INTO "problems" ("uuid", "description", "is_done", "owner_uuid", "day_report_uuid")
+VALUES
+('47ff13e9-aedc-47aa-9dad-d219fb92b694', 'this is the biggest problem we have faced so far', false, '1b3d5e7f-5a1e-4d3a-b1a5-d1a1d5b7a7e1', '25ceb64e-7a57-4ce0-a4fd-45982d9fce38');
+
+INSERT INTO "problems_job_tags" ("problem_uuid", "job_tag_uuid")
+VALUES
+('47ff13e9-aedc-47aa-9dad-d219fb92b694', 'c73ff20b-e64e-4e5f-b270-1a40ba1bd81b');
+
+INSERT INTO "comments" ("uuid", "description", "owner_uuid", "day_report_uuid")
+VALUES
+('f6334156-b08b-434b-b983-f8b11e422804', 'this is amazing!!', '1b3d5e7f-5a1e-4d3a-b1a5-d1a1d5b7a7e1', '25ceb64e-7a57-4ce0-a4fd-45982d9fce38');
+
+INSERT INTO "way_collections" ("uuid", "owner_uuid", "name", "type")
+VALUES
+('30dbf616-b055-4d65-8c0c-e6a913fc119a', '1b3d5e7f-5a1e-4d3a-b1a5-d1a1d5b7a7e1', 'new collection', 'sometype');
+
+INSERT INTO "way_collections_ways" ("way_collection_uuid", "way_uuid")
+VALUES
+('30dbf616-b055-4d65-8c0c-e6a913fc119a', 'a2cb5e1b-44df-48d3-b7a1-34f3d7a5b7e2');
+
+INSERT INTO "way_tags" ("uuid", "name")
+VALUES
+('46d5dd00-75fc-4563-9616-5252a6fa05d3', 'some tag');
+
+INSERT INTO "ways_way_tags" ("way_uuid", "way_tag_uuid")
+VALUES
+('a2cb5e1b-44df-48d3-b7a1-34f3d7a5b7e2', '46d5dd00-75fc-4563-9616-5252a6fa05d3');
+
+INSERT INTO "user_tags" ("uuid", "name")
+VALUES
+('8749d799-0a89-4ffd-b1bd-02ada9234e5a', 'some tag');
+
+INSERT INTO "users_user_tags" ("user_uuid", "user_tag_uuid")
+VALUES
+('1b3d5e7f-5a1e-4d3a-b1a5-d1a1d5b7a7e1', '8749d799-0a89-4ffd-b1bd-02ada9234e5a');
+
+INSERT INTO "profile_settings" ("uuid", "pricing_plan", "owner_uuid")
+VALUES
+('07dfa820-a4b2-4a42-b079-405376b240e7', 'pro', '1b3d5e7f-5a1e-4d3a-b1a5-d1a1d5b7a7e1');
+`
+
+func insertTestData(db *sql.DB) error {
+
+	unPrep := strings.Split(testData, ";")
+	queries := unPrep[:len(unPrep)-1]
+	for _, query := range queries {
+		if _, err := db.Exec(query); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// Adds test data
+// @Summary adds test data
+// @Description adds test data for testing purposes
+// @Tags test
+// @Accept json
+// @Produce json
+// @Success 200
+// @Router /test/data [post]
+func AddTestData(db *sql.DB) gin.HandlerFunc {
+
+	return func(ctx *gin.Context) {
+		util.HandleErrorGin(ctx, insertTestData(db))
+		ctx.JSON(http.StatusOK, gin.H{"status": "success"})
+	}
+}
+
+// Resets all data
+// @Summary resets all data
+// @Description resets all data in the database for testing purposes
+// @Tags test
+// @Accept json
+// @Produce json
+// @Success 200
+// @Router /test/data [delete]
+func ResetData(q *db.Queries) gin.HandlerFunc {
```

Comment:
one public endpoint with  post method and no parameters which calls resetDb, and responsible for cleaning and data population should be enough for now